.env

```diff
@@ -33,3 +33,7 @@ REACT_APP_LIMIT_ORDER_API_READ=https://limit-order.kyberswap.com/read-ks/api
 # Kyber DAO Apis
 REACT_APP_KYBER_DAO_STATS_API=https://kyberswap-dao-stats.kyberengineering.io
 REACT_APP_NOTIFICATION_IGNORE_TEMPLATE_IDS=2,29
+
+# oauth
+REACT_APP_ENV=development
```

Comment:
why do we need this?

Reply:
to select which mode sdk can use, for test only. when we officially public this SDK. I will remove it

Reply:
okay, you can add a TODO on it